please

```diff
@@ -22,7 +22,7 @@ fi
 
 # Detect OSX
 SHASUM="sha256sum"
-NETWORK_OPTIONS='--network="host"'
+NETWORK_OPTIONS=--network="host"
```

Comment:
I think it would be better to remove the quotes around `host` rather than around the entire value (although, looking at the surrounding lines, `"` seems to be the preferred quote). Both forms are equivalent (as is the one without any quotes), but I think the variant with quotes around the entire value make it clearer that the quotes are quoting, rather than part of the value (which is, I guess, the confusion that lead to the second set of quotes being added.